src/index.html

```diff
@@ -6,359 +6,311 @@
           <li>
-            <a href="https://twitter.com/wix" target="_blank">Twitter</a>
+            <div><i></i></div>
+            <div>BBQ</div>
           </li>
           <li>
-            <a href="https://plus.google.com/+Wix" target="_blank"
-              >Google Plus</a
-            >
+            <div><i></i></div>
+            <div>Party</div>
           </li>
           <li>
-            <a href="https://www.facebook.com/wix" target="_blank">Facebook</a>
+            <div><i></i></div>
+            <div>Cooking</div>
           </li>
         </ul>
-        <p>Follow me on social netwroks</p>
+      </li>
+    </ul>
+
+    <!-- Skill Section-->
+    <div class="skills-section">
+      <h2>SKILLS</h2>
+      <div class="skill-inner-width">
+        <div class="skill-left">
+          <div class="skill" id="msexcel">
+            <p class="skill_name">Microsoft Excel</p>
+            <p class="skill_level">Advanced</p>
+          </div>
+          <div class="skill" id="msword">
+            <p class="skill_name">Microsoft Word</p>
+            <p class="skill_level">Advanced</p>
+          </div>
+          <div class="skill" id="adobe">
+            <p class="skill_name">Adobe</p>
+            <p class="skill_level">Proficient</p>
+          </div>
+          <div class="skill" id="ppt">
+            <p class="skill_name">PowerPoint</p>
+            <p class="skill_level">Proficient</p>
+          </div>
+          <div class="skill" id="french">
+            <p class="skill_name">French</p>
+            <p class="skill_level">Native</p>
+          </div>
+        </div>
+        <div class="skill-right">
+          <div class="skill" id="spanish">
+            <p class="skill_name">Spanish</p>
+            <p class="skill_level">Advanced</p>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="contacts-section">
+      <section class="quick-id">
+        <h2>QUICK ID</h2>
+        <table>
+          <tr>
+            <th class="quick-id-label">Phone</th>
+            <th class="quick-id-content">123-456-7890</th>
+          </tr>
+          <tr>
+            <th class="quick-id-label">Email</th>
+            <th class="quick-id-content">
+              <a href="mailto:info@mysite.com">info@mysite.com</a>
+            </th>
+          </tr>
+          <tr>
+            <th class="quick-id-label">Website</th>
+            <th class="quick-id-content">
+              <a href="" target="_blank">www.wix.com</a>
+            </th>
+          </tr>
+          <tr>
+            <th class="quick-id-label">Address</th>
+            <th class="quick-id-content">
+              <address>
+                500 Terry Francois Street San Francisco, CA 94158
+              </address>
+            </th>
+          </tr>
+        </table>
+      </section>
+      <section class="contact-me">
+        <h2>CONTACT ME</h2>
+        <form id="contact-me-form" action="/" method="post">
+          <label for="name"
```

Comment:
wrap an empty label around the input won't help improve accessibility... screen readers can't find any label text to read.

Reply:
Please refer to the modal answer for an example in the remote master branch